src/Commands/BaseCommand.php

```diff
@@ -87,7 +87,7 @@ protected function getFileContent(InputInterface $input, $option)
             return $filename;
         }
 
-        if ($filename && ftell(STDIN) === 0) {
+        if ($filename && ftell(STDIN)) {
```

Comment:
Why this? `ftell()` returns the position of the pointer. I'd be happy to use `ftell(STDIN) !== false` if you were looking for more type-safety here.

Reply:
I agree)